fbchat/models.py

```diff
@@ -436,6 +439,87 @@ def __repr__(self):
     def __unicode__(self):
         return '<Mention {}: offset={} length={}>'.format(self.thread_id, self.offset, self.length)
 
+class Poll(object):
+    #: ID of the poll
+    uid = None
+    #: Title of the poll
+    title = None
+    #: List of :class:`PollOption`
+    options = None
+    #: Options count
```

Comment:
Maybe an added note here, stating that all options may not always be available, and that they can be fetched with `Client.fetchPollOptions`

Reply:
Done